lib/identifiers/doi.rb

```diff
@@ -1,43 +1,31 @@
 module Identifiers
   class DOI
     REGEXP = %r{
-      \b
-      10 # Directory indicator (always 10)
+      10                                      # Directory indicator (always 10)
```

Comment:
By removing the `\b`, this will also match strings that don't start with 10. For example: `110.1097/01.ASW.0000443266.17665.19`.

Reply:
I'll add a test for it (this needs porting to the PHP version too as it was missing there).